### PR TITLE
Adding WOFF2 support

### LINKED ISSUE #53: Woff2 support for the font
It would be great if the custom font could be provided in woff2 format as well, as it is supported by most browsers and is more efficient than woff (and so should be the preferred format for supporting browsers)

### PR DESCRIPTION
**Pull request by S1SYPHOS**
Fixes #53.
Also correcting typos in README.md

Additionally, please update dist/css after the merge, so people using bower etc may profit from the changes. If there's anything else, let me know, and thanks for this great library.
Cheers!

### CODE REVIEW

File: src/sass/lg-fonts.scss [hunk 1]
```diff
@@ -1,8 +1,12 @@
 // font icons support
 @font-face {
     font-family: 'lg';
-    src: url("#{$lg-path-fonts}/lg.eot?n1z373");
-    src: url("#{$lg-path-fonts}/lg.eot?#iefixn1z373") format("embedded-opentype"), url("#{$lg-path-fonts}/lg.woff?n1z373") format("woff"), url("#{$lg-path-fonts}/lg.ttf?n1z373") format("truetype"), url("#{$lg-path-fonts}/lg.svg?n1z373#lg") format("svg");
+    src: url("#{$lg-path-fonts}/lg.eot?n1z373"); /* IE9 Compat Modes */
+    src: url("#{$lg-path-fonts}/lg.eot?#iefixn1z373") format("embedded-opentype"), /* IE6-IE8 */
```
Comment (stof) [inline src/sass/lg-fonts.scss:5]: for cache busting to work fine, shouldn't `n1z373` be part of the query string instead (so using `lg.eot?n1z373#iefix` ?

Comment (S1SYPHOS) [reply src/sass/lg-fonts.scss:5]: Frankly, I'm all for dropping it completely .. when do those really ever change anyway 🐃  .. but other apart from that, sure.

**Comment by S1SYPHOS** [comment]
Would love to see this merged ..

**Comment by S1SYPHOS** [comment]
Well, it's been a while .. I'm using WOFF2 in production for all my projects, so merging this would definitely be in order! :+1:

**Comment by S1SYPHOS** [comment]
Well, it is .. @sachinchoolur

**Comment by S1SYPHOS** [comment]
Closing this in favour of https://github.com/sachinchoolur/lightGallery/pull/1030

**Pull request closed without merging**